pages/bridge.tsx

```diff
@@ -245,13 +245,13 @@ const Bridge = () => {
               <div className="mt-5">
                 {bridgeTarget === BridgeTarget.L2
                   ? bridgeToL2Error &&
-                    BigInt(amount) !== BigInt(0) && (
+                    parseEther(amount) !== BigInt(0) && (
                       <ErrorBox heading="There's a problem simulating your bridge transaction:">
                         {bridgeToL2Error.cause?.toString() || bridgeToL2Error.message}
                       </ErrorBox>
                     )
                   : bridgeToL1Error &&
-                    BigInt(amount) !== BigInt(0) && (
+                    parseEther(amount) !== BigInt(0) && (
```

Comment:
see above comment

Reply:
Changed to `rawAmount` in the same commit mentioned above: [abbee38](https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/90/commits/abbee389c617ff9cb42ff93ea301963e1c1c4bf1)